### ISSUE TITLE
Add xgboost to requirements?

### ISSUE THREAD

**Issue by louisdorard** [yassineAlouini/hyperparameters-optimization-talk#5]

**Comment by yassineAlouini** [comment]
Indeed, it is missing. Thanks for pointing this @louisdorard. I have fixed it. Let me know if it is fine now and I will close the issue.

**Comment by louisdorard** [comment]
Thanks @yassineAlouini 
I've been stuck for a couple of hours on "Collecting xgboost==0.6a2 (from -r requirements.txt (line 64))" after rerunning 

> pip install -r requirements.txt

Not sure how to best install xgboost... I'd tried via pip in the past and it had failed. Eventually I created a separate env for it and installed with 

> conda install -c derickl xgboost

**Comment by yassineAlouini** [comment]
I am sorry for the troubles. I guess you use a recent version of OSX. I think there is an issue on the XGBoost repo about this here: https://github.com/dmlc/xgboost/issues/1501. 

Otherwise, you could try to install a less recent version (0.4a for example). 

Next time, I will provide a Docker image. It will make things smoother for everyone. 

Sorry again.

**Comment by louisdorard** [comment]
Hey Yassine, just wondering if you'd been able to look into providing a Docker file or image?

**Comment by yassineAlouini** [comment]
That would be a great idea. I will look into it in the upcoming weeks (I will be busy the next two weeks). Will let you know once it is ready.
